src/Sulu/Bundle/WebsiteBundle/EventSubscriber/AnalyticsSerializeEventSubscriber.php

```diff
@@ -50,5 +50,11 @@ public function onPostSerialize(ObjectEvent $event)
             $value->domains = true;
             $event->getVisitor()->visitProperty($metadata, $value, $event->getContext());
         }
+
+        $content = $analytics->getContent();
```

Comment:
Had a look at it, somehow it is a wrong cache value. The serializer will cache the value based on the first object it gets. And the value differs from object to object (which I would already consider bad design). It should always be an object, also if only a key is stored, just to be consistent. I would say we add a comment here and create a new issue for that tagged on 2.0, so that we can fix that properly then.